### ISSUE TITLE
Verbose errors after failed run

### ISSUE THREAD

**Issue by anarazel** [PGBuildFarm/client-code#14]
I've a number of VPATH using buildfarm animal setup. Whenever one branch fails, run_build.pl spill a lot of these to stderr:
```
cannot unlink file for inst/data-C/base/64190/3395: No such file or directory at ./run_build.pl line 519.
cannot restore permissions to 0100600 for inst/data-C/base/64190/3395: No such file or directory at ./run_build.pl line 519.
cannot unlink file for inst/data-C/base/64190/2613_vm: No such file or directory at ./run_build.pl line 519.
cannot restore permissions to 0100600 for inst/data-C/base/64190/2613_vm: No such file or directory at ./run_build.pl line 519.
cannot unlink file for inst/data-C/base/64190/2657: No such file or directory at ./run_build.pl line 519.
```
The relevant line is:
```
            rmtree("inst") unless $keepall;
```

**Comment by adunstan** [comment]
Do your animals use wait_timeout? if so this is probably fixed by commit 89456b7.

**Comment by adunstan** [comment]
closing since I believe this has now been fixed
